src/main/java/seedu/address/model/person/Person.java

```diff
@@ -31,6 +32,7 @@ public Person(Name name, Phone phone, Email email, Address address) {
         requireAllNonNull(name, phone, email, address);
         this.name = name;
         this.phone = phone;
+        this.nokPhone = new NokPhone();
```

Comment:
Based on my understanding, this constructor is used for add command and above 4 fields **plus** the newly added `NokPhone` field must be present in order to add the person to the record. I think this is also consistent with what @sjoann previously mentioned. Maybe we can add `NokPhone` to the constructor parameter?

Reply:
I have learned that this will be included in the future PR, hence this comment is temporarily resolved.